packages/sdf-cli-local/CONTRIBUTING.md

```diff
@@ -0,0 +1,12 @@
+# Contributing to CLI for Node.js
+
+*Copyright (c) 2014, 2019 Oracle and/or its affiliates The Universal Permissive License (UPL), Version 1.0*
+
+**Pull requests are currently not being accepted for the CLI for Node.js project.**
```

Comment:
what if we update it to:
```
# Contributing to @oracle/netsuite-suitecloud-localserver-cli-command extension 

*Copyright (c) 2014, 2019 Oracle and/or its affiliates The Universal Permissive License (UPL), Version 1.0*

**Pull requests are currently not being accepted for @oracle/netsuite-suitecloud-localserver-cli-command extension.**
```
So we put the name of the package/extension instead just Node.js CLI ?

Reply:
I am not sure we need CONTRIBUTING.md on the package level. PRs are created on repository level anyways so maybe we just need one file in the root.

Reply:
Right. Single file for the repo makes sense.
The license file should definitely be in every package since it should be included when the customer does `npm install` of a specific package.